.github/actions/start-build/action.yml

```diff
@@ -37,7 +37,7 @@ runs:
     shell: bash
     run: |
       # pinned pip to 21.0a and setuptools<58.0.0 as current dependencies do not build on newer versions of pip and setuptools
-      pip install --upgrade pip==21.0.1 "setuptools<58.0.0" "wheel==0.38.4"
+      pip install --upgrade pip==24.0.0 setuptools
```

Comment:
Should we still ceiling or ping setup-tools with a higher version?

Reply:
I wouldn't say so, the only reason for `setuptools` ceiling was that old `python3.6` dependencies refused to install on `python 3.10` without `setuptools` constrained `<58.0.0`